client/verta/tests/unit_tests/registry/test_check_model_dependencies.py

```diff
@@ -49,28 +41,30 @@ def test_check_model_dependencies_complete(dependency_testing_model, complete_en
 
 
 def test_check_model_dependencies_missing_raise(dependency_testing_model, complete_env) -> None:
-    """ Verify that check_model_dependencies raises an exception for missing
-    packages when `raise_for_missing` is True.
+    """ Verify that check_model_dependencies raises an exception, with the
+    correct message, for missing packages when `raise_for_missing` is True.
     """
     incomplete_env = Python(
-        [ r for r in complete_env.requirements if r != 'click==0.0.1' ]
+        [r for r in complete_env.requirements if r != 'click==0.0.1']
     )  # drop a single dependency to be caught
     with pytest.raises(RuntimeError) as err:
         _check_model_dependencies(
             model_cls=dependency_testing_model,
             environment=incomplete_env,
             raise_for_missing=True,
         )
-    assert err.value.args[0] == "the following packages are required by the model " \
-                                "but missing from the environment: {'click'}"
+    assert err.value.args[0] == "the following import modules are required by the model " \
+                                "but missing any corresponding distribution package in the" \
+                                " environment:\n{'import_module': 'click', " \
+                                "'distribution_packages': ['click']}"
```

Comment:
I don't think users (especially data scientists) are going to know what a "distribution package" or "import module" is.

I'd recommend just saying "the following packages are required by the model but missing from the environment" and list just the distribution package names (concatenating all the namespace packages, because that's the best we can do for now).

Or if we have to get in-depth, something like "yaml (installed via ['PyYAML']), pandas (installed via ['pandas'])".

Reply:
Updated